File: lingting-core/src/main/java/live/lingting/framework/exception/HttpException.java
package live.lingting.framework.exception;

/**
 * @author lingting 2024-01-29 16:05
 */
public class HttpException extends RuntimeException {

	public HttpException(String message) {
		super(message);
	}

	public HttpException(String message, Throwable cause) {
		super(message, cause);
	}

	public HttpException(Throwable cause) {
		super(cause);
	}

}

File: lingting-core/src/main/kotlin/live/lingting/framework/exception/HttpException.kt
package live.lingting.framework.exception

/**
 * @author lingting 2024-01-29 16:05
 */
class HttpException : RuntimeException {
    constructor(message: String?) : super(message)

    constructor(message: String?, cause: Throwable?) : super(message, cause)

    constructor(cause: Throwable?) : super(cause)
}
